.github/workflows/pr_tests.yml

```diff
@@ -84,21 +84,35 @@ jobs:
       matrix:
         machines: [ 1, 2, 3, 4 ]
 
-    env:
-      BASIC_AUTH_KEY: ${{ secrets.BASIC_AUTH_KEY }}
-      BASIC_AUTH_SECRET: ${{ secrets.BASIC_AUTH_SECRET }}
-      MULESOFT_API_CIS_URL: ${{ secrets.MULESOFT_API_CIS_URL }}
-      MULESOFT_API_CIS_KEY: ${{ secrets.MULESOFT_API_CIS_KEY }}
-      MULESOFT_API_CIS_SECRET: ${{ secrets.MULESOFT_API_CIS_SECRET }}
-      MULESOFT_API_MDM_EA_URL: ${{ secrets.MULESOFT_API_MDM_EA_URL }}
-      MULESOFT_API_MDM_EA_KEY: ${{ secrets.MULESOFT_API_MDM_EA_KEY }}
-      MULESOFT_API_MDM_EA_SECRET: ${{ secrets.MULESOFT_API_MDM_EA_SECRET }}
-      GOOGLE_ANALYTICS_ID: ${{ secrets.GOOGLE_ANALYTICS_ID }}
+    # Cancel Previous runs
+    # concurrency:
+    #   group: e2e-${{ github.workflow }}-${{ github.ref }}-${{ matrix.containers }}
+    #   cancel-in-progress: true
 
     steps:
       - uses: actions/checkout@v3
       - name: Start Docker Compose
+        env:
+          API_URL: ${{ secrets.API_URL }}
```

Comment:
wasn't this added/moved in a different PR?

Reply:
Yes, that PR was merged into this PR instead of `main-application` unfortunately